denizen_scripts/HeroCraft/mobs/prefixes/shrieking.dsc

```diff
@@ -0,0 +1,7 @@
+custom_mob_prefix_shrieking:
+  Type: world
+  debug: false
+  events:
+    after entity_flagged:shrieking damages entity:
+        - stop if:<util.random_chance[95]>
```

Comment:
5% is REALLY HIGH, move this to a suffix

Reply:
k